Source/PTEDashboard.m

```diff
@@ -40,7 +40,7 @@ + (PTEDashboard *)sharedDashboard
     dispatch_once(&onceToken, ^
                   {
                       CGRect frame = UIScreen.mainScreen.bounds;
-                      _sharedDashboard = [[self alloc] initWithFrame:frame];
+                      _sharedDashboard = [[self alloc] initWithFrame:UIEdgeInsetsInsetRect(window.frame, window.safeAreaInsets)];
```

Comment:
![image](https://user-images.githubusercontent.com/1261982/68025322-d7eb9780-fc82-11e9-8c45-87445aa0eec6.png)

Reply:
The line above should be changed to 

```
UIWindow *window = UIApplication.sharedApplication.keyWindow;
```